content/docs/jsx-in-depth.md

```diff
@@ -245,7 +245,31 @@ function App2() {
 }
 ```
 
-Spread attributes can be useful when you are building generic containers. However, they can also make your code messy by making it easy to pass a lot of irrelevant props to components that don't care about them. We recommend that you use this syntax sparingly.
+You can also pick specific props that your component will consume while passing all other props using the spread operator. 
+This ensures that the component consumes the `kind` prop only, and passes down all other props via `...other`.
```

Comment:
nit: I think this sentence may not be necessary. I think it might be nice to just show the example and then explain it below (like you already do) 👍